docusaurus/docs-internals/editor-controls/multivalue-controls/transform.md

```diff
@@ -84,6 +112,42 @@ Adding a label on the left side of the control.
 }
 ```
 
+#### Class name example
+
+Adding a CSS class to the control's DOM node.
+
+```js
+{
+  id: "effect",
+  type: "transform"
+  className: "brz-transform"
+}
+```
+
+#### Icon example
+
+Adding a "repeat" icon to the left of the control's label.
```

Comment:
you add `"nc-cog"` icon instead of "repeat"

replace with : "effects" and "nc-effects"